src/landing/home-stats/home-stats-table.jsx

```diff
@@ -211,3 +259,16 @@ function getLabelForStatKey(key) {
       return localise('Total Users', {});
   }
 }
+
+/**
+ * @param {import('ag-grid-community').ICellRendererParams} params
+ * 
+ * @returns {JSX.Element | string}
+ */
+function HandleLinkRenderer(params) {
+    try{
+      return (<a href={new URL(params.data.ProfileURL).href} target="_blank">{params.data.Handle}</a>);
```

Comment:
do we want to link up bsky or their profile on clearsky?

Reply:
Yeah, linking this to their profile on ClearSky makes sense. I can handle this in another PR.